ajax/libs/css-element-queries/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "css-element-queries",
+  "filename": "ElementQueries.min.js",
+  "version": "0.4.0",
+  "description": "CSS-Element-Queries Polyfill. proof-of-concept for high-speed element dimension/media queries in valid css.",
+  "keywords": [
+    "css",
+    "element",
+    "query"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:marcj/css-element-queries.git"
+  },
+  "author": "Marc J. Schmidt",
+  "license": "MIT",
+  "homepage": "https://marcj.github.io/css-element-queries/",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/marcj/css-element-queries.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "+(ElementQueries*|ResizeSensor*)"
```

Comment:
You can use `"+(ElementQueries|ResizeSensor)*"` to make it more easier, though it can also work. It's just a suggestion.